docs/vm-installation.md

```diff
@@ -0,0 +1,115 @@
+# Installing antrea-agent on VM
+
+`antrea-agent` can be run on Linux/Windows VMs. Below are the steps needed to configure and run
```

Comment:
How about "This document talks about the steps..."

Reply:
Changing it to: This document lists/describes the steps needed to configure and run